server/routes/send_message.test.ts

```diff
@@ -156,20 +206,22 @@ describe('send_message route when rootAgentId is provided', () => {
     })) as ResponseObject;
     expect(mockedLogger.error).toBeCalledWith(new Error('something went wrong'));
     expect(result.source).toMatchInlineSnapshot(`
-        Object {
-          "interactions": Array [],
-          "messages": Array [],
-          "sessionId": "foo",
-          "title": "foo",
-        }
-      `);
+      Object {
+        "interactions": Array [
+          undefined,
+        ],
+        "messages": undefined,
```

Comment:
Shall we return an empty array instead of `undefined`?

Reply:
Sure and it need a better error handling when return interactions and messages. Done